File: Source/Web/Scripts/App/Models/Invoice.ts
/// <reference path="../exceptionless.ts" />

module exceptionless.models {
    export class Invoice {
        id: string;
        date: Date;
        paid: boolean;

        constructor(id: string, date: Date, paid: boolean) {
            this.id = id;
            this.date = date;
            this.paid = paid;

            ko.track(this);
        }
    }
}

File: Source/Api/Models/Organization/Invoice.cs
using System;
using System.Collections.Generic;

namespace Exceptionless.Api.Models {
    public class Invoice {
        public Invoice() {
            Items = new List<InvoiceLineItem>();
        }

        public string Id { get; set; }
        public string OrganizationId { get; set; }
        public string OrganizationName { get; set; }

        public DateTime Date { get; set; }
        public bool Paid { get; set; }
        public double Total { get; set; }

        public IList<InvoiceLineItem> Items { get; set; }
    }
}